packages/driver/cypress/e2e/commands/net_stubbing.cy.ts

```diff
@@ -2540,11 +2536,9 @@ describe('network stubbing', function () {
         })
       })
 
-      it('when body contains unicode', function (done) {
+      it('when body contains unicode', function () {
         cy.intercept('/post-only', function (req) {
           expect(req.headers['content-length']).to.eq('8')
-
-          done()
```

Comment:
Fixes https://github.com/cypress-io/cypress/issues/23150. We didn't need `done()` here -- passing and failing test cases work as expected here without calling `done()`